connector/MongoDBAtlasODBC.pq

```diff
@@ -204,20 +204,7 @@ MongoDBAtlasODBC.Icons = [
 
 BuildOdbcConfig = () as record =>
     let
-        AstVisitor = [

```

Comment:
umh, being the devil's advocate here, but will this not break our connector since we don't support TOP?

https://learn.microsoft.com/en-us/power-query/odbc-parameters#overriding-astvisitor
>  Note
Drivers that support LIMIT and OFFSET clauses (rather than TOP) will want to provide a LimitClause override for AstVisitor.

Reply:
We do specify `LimitClauseKind = LimitClauseKind.LimitOffset,` ([link](https://learn.microsoft.com/en-us/powerquery-m/limitclausekind-type)) .  Which may be enough according to their comment in the email.  I implemented the AstVisitor when I saw the same `Note` that you linked.